californium-core/src/main/java/org/eclipse/californium/core/coap/Option.java

```diff
@@ -169,7 +185,9 @@ public String getStringValue() {
 	}
 	
 	/**
-	 * Gets the option value as integer.
+	 * Gets the option value as integer. Handles cases where {@code value}
+	 * contains leading 0's or a case where {@code value} is empty which
+	 * returns 0.
```

Comment:
Though this applies also to `getLongValue()`, I would prefer to have it add also there.

Reply:
Good point, I'll add something there.